measures/ResStockArguments/measure.rb

```diff
@@ -465,7 +475,8 @@ def run(model, runner, user_arguments)
 
     # Other
     if args[:misc_plug_loads_other_annual_kwh].to_s == Constants.Auto
-      if [HPXML::ResidentialTypeSFD].include?(args[:geometry_unit_type])
+      # FIXME: Need to disaggregate detached and mobile home
+      if [HPXML::ResidentialTypeSFD, HPXML::ResidentialTypeManufactured].include?(args[:geometry_unit_type])
```

Comment:
Probably not worth changing this right now. I'm looking to incorporate these plug load equations into OpenStudio-HPXML, and will separate out SFD and MH there.

Reply:
https://github.com/NREL/OpenStudio-HPXML/pull/1775